scripts/release/rpm/azure-cli.spec

```diff
@@ -5,11 +5,6 @@
 # Turn off python byte compilation
 %global __os_install_post %(echo '%{__os_install_post}' | sed -e 's!/usr/lib[^[:space:]]*/brp-python-bytecompile[[:space:]].*$!!g')
 
-# .el7.centos -> .el7
-%if 0%{?rhel}
-  %define dist .el%{?rhel}
-%endif
-
```

Comment:
In CentOS 7, RHEL 8.4, RHEL 9.0.0, it is `.el7/8/9`.
The default values are correct, we can remove this line.

PS: In RHEL 7.9 `%{dist}` is `.el7_9`.
Before CentOS 7.5, `%{dist}` is `.el7.centos`, ref: https://gist.github.com/seemethere/7fde5973d54355ed4690652d8a51829d